grouper-misc/webapp/pom.xml

```diff
@@ -38,12 +38,17 @@
             <artifactId>grouper-ui</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>grouper-ws</artifactId>
+            <version>${project.version}</version>
+        </dependency>
     </dependencies>
 
     <build>
         <resources>
             <resource>
-                <directory>../../../grouper/conf</directory>
+                <directory>../../grouper/conf</directory>
```

Comment:
Take this out

Reply:
I think I remember why this is here, but need to verify: ddls for some reason were missing from the classpath

Reply:
DDLs should be packaged in the grouper jar now